apps/ai-content-generator/src/components/app/sidebar/SidebarButtons.tsx

```diff
@@ -3,18 +3,8 @@ import featureConfig, { AIFeature } from '@configs/features/featureConfig';
 import FeatureButton from './feature-button/FeatureButton';
 
 const SidebarButtons = () => {
-  const featureList = Object.entries(featureConfig).map(([featureId, configItem]) => {
-    const { helpText, buttonTitle, title } = configItem;
-
-    return (
-      <FeatureButton
-        key={featureId}
-        text={buttonTitle}
-        dialogTitle={title}
-        helpText={helpText}
-        feature={featureId as AIFeature}
-      />
-    );
+  const featureList = Object.entries(featureConfig).map(([featureId]) => {
+    return <FeatureButton key={featureId} feature={featureId as AIFeature} />;
```

Comment:
So much easier to look at now!

You can change it from `Object.entries` to just `Object.keys` since we only need the key now!
```suggestion
  const featureList = Object.keys(featureConfig).map((featureId: AIFeature) => {
    return <FeatureButton key={featureId} feature={featureId} />;
```

Reply:
makes complete sense! I was wondering if key would accept an enum as I was typing it